app/services/frenet/handler.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Frenet
+  class Handler
+    def initialize(shop)
+      @shop = shop
+    end
+
+    def shipping_info
+      response = Excon.get(
+        'http://api.frenet.com.br/shipping/info',
+        headers: headers
+      )
+      sentry_extras(response)
+      JSON.parse(response.body).dig('ShippingSeviceAvailableArray')
+    end
+
+    def tracking(body)
+      response = Excon.post(
+        'http://api.frenet.com.br/tracking/trackinginfo',
+        body: body.to_json,
+        headers: headers
+      )
+      sentry_extras(response)
+      response.body
```

Comment:
Poderia já retornar o HASH assim como o anterior.

```suggestion
      JSON.parse(response.body)
```

Reply:
Alteração feita